tex/chapters/07-ApplyingModel.tex

```diff
@@ -0,0 +1,197 @@
+%!TEX root = ../main.tex
+
```

Comment:
One thing missing here: so what is the model?

If I want to just use the model I don't want to have to train it again if that makes sense. If we were going with the logistic regression this would mean writing the coefficients of each variable here. I don't know enough about the model you've ended up going with to say what the equivalent is.

There's stuff about picking or similar sklean models which is also something you can do but we ideally want you to "write down the model" here too. (before the p values actually).